Module/templates/bug-report.html

```diff
@@ -13,6 +13,11 @@ <h2 class="flexrow">
 </div>
 
 {{#if useBugReporter}}
+  <!--If game version is supported, hide this -->
+  <div class="{{#if unsupportedCore}}{{else}}hidden{{/if}} notification warning">
```

Comment:
```suggestion
  {{#unless unsupportedCore}}<div class="notification warning">
```
No need to create this DOM element unless we're in an unsupported core version, which we know from the start via getData. `hidden` class is good for async flows which we don't necessarily know about from the getgo.

Reply:
Totally forgot about # unless